### PR TITLE
release 0.4.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ Contribute
...
 - [Github][]
 
+
+Making a new Release
+--------------------
+
+To make a new release, you need to generate changelog:
+
+```
+bundle intall --path .vendor/
```
Comment (towo) [inline README.md]: typo

Comment (towo) [reply README.md]: also "generate a changelog"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# Changelog
```
Comment (rski) [inline CHANGELOG.md]: there already is a CHANGES.md file i n the repo, it should be taken into account before merging this PR (rename changes to CHANGELOG?)

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+- incorrect indentation when using -\> [\#65](https://github.com/voxpupuli/puppet-mode/issues/65)
+- Set require-final-line to the value of mode-require-final-line, not t [\#52](https://github.com/voxpupuli/puppet-mode/issues/52)
+- Trailing colon in a comment causes an indent [\#49](https://github.com/voxpupuli/puppet-mode/issues/49)
+- Applying manifests over tramp [\#46](https://github.com/voxpupuli/puppet-mode/issues/46)
+- Regexp font-locking issue [\#42](https://github.com/voxpupuli/puppet-mode/issues/42)
+- Array indentation seems broken [\#33](https://github.com/voxpupuli/puppet-mode/issues/33)
+- Add pcase fix for emacs 24.1 [\#50](https://github.com/voxpupuli/puppet-mode/pull/50) ([kmosher](https://github.com/kmosher))
+
+**Closed issues:**
```
Comment (rski) [inline CHANGELOG.md]: Admittedly i've fallen off the wagon with the changelog generator thing, but this seems like it should be in the ehancements section. Who decides which changes should be in which section?

**Comment by jacobisaliveandwell** [comment]
@bastelfreak @towo Any way to move this along?

**Pull request merged**